src/components/Atoms/Controls/Input/types/types.ts

```
@@ -0,0 +1,35 @@
+import { MenuItem } from "@/types";
+import { RegisterOptions } from "react-hook-form";
+
+export interface CustomChangeEvent {
+  target: {
+    name: string;
+    value: any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)